### PR TITLE
Changes for App Service Editor in App Menu

### PR DESCRIPTION
**Pull request by prashanthmadi**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@prashanthmadi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by PRmerger** [comment]
@prashanthmadi : Thanks for your contribution to the Azure documentation! The author, @christopheranderson, has been notified to review your proposed change.

**Comment by christopheranderson** [comment]
@lindydonna - Could you review these since you're the App Service Editor expert?

File: articles/azure-functions/functions-reference.md [hunk 1]
```diff
@@ -78,21 +78,19 @@ The function editor built into the Azure portal lets you update the *function.js
...
 
-3. Click **Tools**.
+3. Click **App Service Editor** in App Menu Nav under **DEVELOPMENT TOOLS**.
 
-4. Under **Develop**, click **Visual Studio Online**.
-
-5. Turn it **On** if it is not already enabled, and click **Go**.
+4.  click **Go**.
 
 	After Visual Studio Online loads, you'll see the *host.json* file and function folders under *wwwroot*.
```
Comment (lindydonna) [inline articles/azure-functions/functions-reference.md]: @prashanthmadi Thanks for your contribution!  Do you mind also changing the reference on line 91?  If not, we'll accept the PR and fix ourselves :)

Comment (prashanthmadi) [inline articles/azure-functions/functions-reference.md]: @lindydonna , yes i have made that change.

**Comment by lindydonna** [comment]
#sign-off

**Comment by PRmerger** [comment]
Only the author of the article can use the sign off comment to add the ready-to-merge label.

**Comment by christopheranderson** [comment]
#sign-off

**Pull request merged**